compiler/rustc_lint/src/lints.rs

```diff
@@ -2643,7 +2641,7 @@ impl<G: EmissionGuarantee> LintDiagnostic<'_, G> for ElidedNamedLifetime {
         //  but currently this lint's suggestions can conflict with those of `clippy::needless_lifetimes`:
         //  https://github.com/rust-lang/rust/pull/129840#issuecomment-2323349119
         // HACK: `'static` suggestions will never sonflict, emit only those for now.
-        if name != rustc_span::symbol::kw::StaticLifetime {
+        if name != rustc_span::kw::StaticLifetime {
```

Comment:
Question: can this just also import `kw` or is `kw` used as a local var here somewhere?

Reply:
:+1: